src/core/unicode.js

```diff
@@ -1630,7 +1630,7 @@ var getNormalizedUnicodes = getArrayLookupTableFactory(function () {
 
 function reverseIfRtl(chars) {
   var charsLength = chars.length;
-  // Reverse an arabic ligature.
```

Comment:
Please don't remove existing comments!

Reply:
Sorry! Could I please move the comment up to line 1630 and change it to say `// Reverse an RTL (right-to-left) ligature`? I removed it because the line it's above DOES NOT actually reverse a ligature, and the method as a whole deals with other RTL languages in addition to Arabic